processing/src/main/java/io/druid/query/extraction/MapLookupExtractor.java

```diff
@@ -43,12 +43,16 @@
 {
   private final Map<String, String> map;
 
+  private final boolean isOneToOne;
+
   @JsonCreator
   public MapLookupExtractor(
-      @JsonProperty("map") Map<String, String> map
+      @JsonProperty("map") Map<String, String> map,
+      @JsonProperty("isOneToOne") boolean isOneToOne
```

Comment:
@jon-wei and @drcrallen i think it is more clear to use `OneToOne`:
First the actual property of extraction type is already called `ExtractionType.ONE_TO_ON`
Second i think  `OneToOne` is more clear than the mathematical term injective.